libs/wire-api/src/Wire/API/Team/Feature.hs

```diff
@@ -298,6 +298,7 @@ instance FromByteString TeamFeatureTTLValue where
 instance Cass.Cql TeamFeatureTTLValue where
   ctype = Cass.Tagged Cass.IntColumn
 
+  fromCql (Cass.CqlInt 0) = pure TeamFeatureTTLUnlimited
```

Comment:
would this have been really bad (unlimited enablement would have been disabled immediately again), or no problem (0 means "indefinite" in cassandra as well)?  either way, it should be fixed, and we should have roundtrip tests for this, and there should be an error if a cql instance does not have a test, and...